src/main/java/org/jenkinsci/plugins/ghprb/extensions/status/GhprbSimpleStatus.java

```diff
@@ -165,6 +174,11 @@ public void onBuildStart(AbstractBuild<?, ?> build,
         } else {
             sb.append(Ghprb.replaceMacros(build, listener, startedStatus));
         }
+
+        if(showMatrixStatus && !(build.getProject() instanceof MatrixProject)){
```

Comment:
From my research, I've noticed that a parent job will return true when it is checking if it is an instance of a MatrixProject. However, the child job that is was triggered by the parent job will return false for this condition. Thus, only the child job will enter the if statement and exit before creating a commit status. Both the parent and child job will return true for the showMatrixStatus attribute.